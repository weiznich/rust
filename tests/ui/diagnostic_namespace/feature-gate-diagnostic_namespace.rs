// check-fail
#[diagnostic::non_existing_attribute]
//~^ERROR use of undeclared crate or module `diagnostic
pub trait Bar {
}

#[diagnostic::non_existing_attribute(with_option = "foo")]
//~^ERROR use of undeclared crate or module `diagnostic
struct Foo;

fn main() {
}
