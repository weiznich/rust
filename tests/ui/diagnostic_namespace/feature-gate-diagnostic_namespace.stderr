error[E0433]: failed to resolve: use of undeclared crate or module `diagnostic`
  --> $DIR/feature-gate-diagnostic_namespace.rs:7:3
   |
LL | #[diagnostic::non_existing_attribute(with_option = "foo")]
   |   ^^^^^^^^^^ use of undeclared crate or module `diagnostic`

error[E0433]: failed to resolve: use of undeclared crate or module `diagnostic`
  --> $DIR/feature-gate-diagnostic_namespace.rs:2:3
   |
LL | #[diagnostic::non_existing_attribute]
   |   ^^^^^^^^^^ use of undeclared crate or module `diagnostic`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0433`.
